example/rollup/app.output.js

```diff
@@ -0,0 +1,19 @@
+// https://github.com/maxogden/websocket-stream/blob/48dc3ddf943e5ada668c31ccd94e9186f02fafbd/ws-fallback.js
+
+var ws = null;
+
+if (typeof WebSocket !== 'undefined') {
+  ws = WebSocket;
+} else if (typeof MozWebSocket !== 'undefined') {
+  ws = MozWebSocket;
+} else if (typeof global !== 'undefined') {
+  ws = global.WebSocket || global.MozWebSocket;
```

Comment:
Moreover, checking for global is useless here, because this is a node.js object
> Historically, accessing the global object has required different syntax in different JavaScript environments. On the web you can use window, self, or frames - but in Web Workers only self will work. In Node.js none of these work, and you must instead use global.